lib/style/module_directives.ex

```diff
@@ -0,0 +1,80 @@
+# Copyright 2023 Adobe. All rights reserved.
+# This file is licensed to you under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License. You may obtain a copy
+# of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+# OF ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+defmodule Styler.Style.ModuleDirectives do
+  @moduledoc """
+  Styles up module directives!
+
+  This Style will expand multi-aliases/requires/imports/use and sort the directive within its groups (except `use`s, which cannot be sorted)
+  It also adds a blank line after each directive group.
+
+  Rewrites for the following Credo rules:
+
+    * `Credo.Check.Consistency.MultiAliasImportRequireUse` (force expansion)
+    * `Credo.Check.Readability.AliasOrder`
+    * `Credo.Check.Readability.MultiAlias`
+    * `Credo.Check.Readability.UnnecessaryAliasExpansion`
+
+  This module is more particular than credo for sorting. Notably, it sorts `alias __MODULE__`, whereas Credo allowed
+  that alias intermixed anywhere in a group.
+  """
+  @behaviour Styler.Style
+
+  @directives ~w(use import require alias)a
+
+  def run({{d, _, _} = directive, %{l: left, r: right} = meta}) when d in @directives do
+    {right, directives} = consume_directive_group(d, [directive | right], [])
```

Comment:
I believe this might not be far from being able to group all directives in order: `use/import/alias/require`, it'd just be a matter of grouping each of the directives in place of the `consume_directive_group` call

I'm going to play with it a bit more - the only unknown is what module attributes look like and how i should (or shouldn't) touch them.

Reply:
I also expect that doing the "put all directives in order" rule might really destroy comment interaction, so might be something we should put off until we have a good comments algorithm